### PR TITLE
17th task 2808

### PR DESCRIPTION
**Pull request by Clasher370**
Changed algorithm.

### CODE REVIEW

File: app/views/flashcards/_form.html.erb [hunk 1]
```diff
@@ -5,7 +5,18 @@
   <%= form_tag compare_path do %>
       <%= hidden_field_tag :id, @card.id %>
       <%= hidden_field_tag :session_try, @try %>
+      <%= hidden_field_tag :timer %>
       <%= text_field_tag :user_text %>
       <%= button_tag %>
   <% end %>
 </div>
+<script>
```
Comment (X1ting) [inline app/views/flashcards/_form.html.erb]: use javascript in separate file in javascripts folder

File: app/views/flashcards/_form.html.erb [hunk 2]
```diff
@@ -5,7 +5,18 @@
...
+    var count=0;
```
Comment (X1ting) [inline app/views/flashcards/_form.html.erb]: indentation

File: db/migrate/20170828190019_add_easiness_to_cards.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#
```
Comment (X1ting) [inline db/migrate/20170828190019_add_easiness_to_cards.rb]: Why you add this? For Hound? 
Maybe better fix Hound config for ignore this?

File: app/services/super_memo.rb [hunk 4]
```diff
@@ -0,0 +1,64 @@
+#
```
Comment (X1ting) [inline app/services/super_memo.rb]: Why you add this? For Hound?
Maybe better fix Hound config for ignore this?

File: app/services/super_memo.rb [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    @timer = timer.to_i
+  end
+
+  def algorithm
+    compare_result = compare(@card.original_text, @user_text)
+    quality = quality_of_repetition(compare_result, @timer)
+    update_card(@card, quality)
+    create_notice(@card, @user_text, quality)
+  end
+
```
Comment (X1ting) [inline app/services/super_memo.rb]: Methods below this line are using outside from class?

File: app/assets/javascripts/application.js [hunk 6]
```diff
@@ -14,3 +14,12 @@
 //= require jquery_ujs
 //= require turbolinks
 //= require_tree .
+
+var count=0;
```
Comment (X1ting) [inline app/assets/javascripts/application.js]: application.js it is configuration file, which require libs only. 
You must create your own .js file in javascripts directory and wrote code there.

File: app/assets/javascripts/application.js [hunk 7]
```diff
@@ -14,3 +14,12 @@
...
+
+setInterval(timer, 1000);
+
+function timer() {
+  count = count + 1;
+  document.getElementById("timer").value = count;
```
Comment (X1ting) [inline app/assets/javascripts/application.js]: use jquery

File: app/assets/javascripts/timer.js [hunk 8]
```diff
@@ -0,0 +1,8 @@
+var count=0;
+
+setInterval(timer, 1000);
+
+function timer() {
+    count = count + 1;
```
Comment (X1ting) [inline app/assets/javascripts/timer.js]: indentaaaaatioooon

Comment (X1ting) [inline app/assets/javascripts/timer.js]: you can use count += 1

Comment (Clasher370) [reply app/assets/javascripts/timer.js]: Rubymiiiiiineee ^ ^

**Pull request merged**